cmd/lakectl/cmd/root.go

```diff
@@ -85,7 +85,7 @@ lakectl is a CLI tool allowing exploration and manipulation of a lakeFS environm
 	Version: version.Version,
 }
 
-func getClient() api.ClientWithResponsesInterface {
+func getEnpointWithAuth() (string, *securityprovider.SecurityProviderBasicAuth) {
```

Comment:
why reason to extract this one out?

Reply:
Was done initially in order to give the ability to create a client without response. Once realized creating a new client was unnecessary, it still seemed to be correct to be split into two separate functions. Can reset you think it's better.